tutorials/xr/ar_passthrough.rst

```diff
@@ -0,0 +1,187 @@
+.. _doc_openxr_passthrough:
+
+AR / Passthrough
+================
+
+Augmented Reality is supported through various methods depending on the capabilities of the hardware.
+
+Headsets such as the Magic Leap and glasses such as TiltFive show the rendered result on transparent displays
+allowing the user to see the real world.
+
+Headsets such as the Quest, HTC Elite, and Lynx R1 implement this through a technique called passthrough,
+where cameras record the real world and these images are used as the background on top of which our rendered
+result is used.
+
+.. note::
+
+    The way specifically passthrough is implemented has vast differences between different platforms.
+
+    In Godot 4.3 we have implemented a unified approach that is explained on this help page
+    so you don't need to worry about these differences, the :ref:`XRInterface <class_xrinterface>`
+    implementation is now responsible for applying the correct platform dependent method [#]_.
+
+    For headsets such as the Meta Quest and HTC Elite you will need to use the
+    `OpenXR vendors plugin v3.0.0 <https://github.com/GodotVR/godot_openxr_vendors/releases>`__
+    or later to enable passthrough.
+
+    For backwards compatibility the old API for passthrough is still available but it is recommended
+    to follow the new instructions below.
+
+Environment blend modes
+-----------------------
+
+The way we configure VR or AR functionality is through setting the environment blend mode.
+This mode determines how the (real world) environment is blended with the virtual world.
+
+.. list-table:: Blend modes
+  :widths: 35 65
+  :header-rows: 1
+
+  * - Blend mode
+    - Description
+  * - XR_ENV_BLEND_MODE_OPAQUE
+    - The rendered image is opaque, we do not see the real world. We're in VR mode.
+      This will turn off passthrough if passthrough is used.
+  * - XR_ENV_BLEND_MODE_ADDITIVE
+    - The rendered image is added to the real world. This mode generally applies to AR only devices
+      such as the Hololens where the optics are not able to obscure the real world.
+      This will turn on passthrough if passthrough is used.
+  * - XR_ENV_BLEND_MODE_ALPHA_BLEND
+    - The rendered image is alpha blended with the real world.
+      This is supported on devices that can fully obscure the real world
+      and generally the preferred way to do AR.
```

Comment:
Following up on my comment about terminology confusion, the use of `AR only devices` and `AR` throughout can get confusing to developers new with the apis, capabilities.

My suggestion would be that we use the terms `see-through` devices to refer to devices like Magic Leap, Hololens, and `video passthrough` devices to refer to devices like Quest, HTC Elite; unless OpenXR has another term for how to refer to either types of devices.

Reply:
I agree on the two term, I'll see what I can write better to make good use of those terms.

On a whole though it is important to note that this part of the documentations main goal is to create a foundation where it shouldn't matter whether AR is created with see-through devices or video passthrough devices, so that if you stick to this guide you create an application that works on any AR device.

The moment you start looking at APIs that only work on a subset of devices, or often just work for one vendor, that has to be a conscious choice on the developer. "Is using this feature worth tying myself into this eco system".

Reply:
Ok, rewrote part of this section, let me know if you agree with the changes.